rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
I saw somewhere in the rollup docs that `named` has to be used when the lib exports something like we do.

```js
export default glamorous
export {ThemeProvider}
```
Isn't it correct?